_chapters/chapter_02.md

```diff
@@ -67,6 +67,15 @@ layout: rails_tutorial
 {% endhighlight %}
 {% endsteps %}
 
+{% aside %}
+### Rails is using ruby! 
+
+You might remember [blocks](http://tutorials.jumpstartlab.com/projects/ruby_in_100_minutes.html#6.-blocks) from the Ruby in 100 minutes tutorial you did before you started building your app. 
+
+> Blocks start with the keyword `do` and ends with the keyword `end`. 
+
+{% endaside %}
```

Comment:
👏🏾

Although aside and protips are the rendered the same, I wonder if this should be a protip instead of an aside.